changelog.d/7779.bugfix

```diff
@@ -0,0 +1 @@
+Allow YAML config file to contain None. Useful when config file is fully commented.
```

Comment:
please could you clarify this to say that it is a change to synctl (rather than synapse itself)?

also, there's no need to say "*YAML* config file". It's just the config file to us!